proposal.tex

```diff
@@ -63,5 +85,5 @@ \subsection{Other catalogs}
 LINCC may again help here by coordinating IDACs to provide neighbor tables/services for other catalogs.
 One notion would be to have a the Object catalog at IDACs matched to their local holdings.
 We could consider migrating some of those neighbor catalogs back to USDF Qserv.
+Gaia will be provided as part of the processing - there is no requirement listing catalogs so from a verification perspective we can put this on IDACs
```

Comment:
The crucial core of the verification is showing that it can be done, not that any specific catalog has actually been ingested and made available to all users.